test/e2e/util.go

```diff
@@ -236,7 +236,7 @@ func deleteEtcdCluster(t *testing.T, f *framework.Framework, e *spec.EtcdCluster
 }
 
 func waitResourcesDeleted(t *testing.T, f *framework.Framework, e *spec.EtcdCluster) error {
-	err := retryutil.Retry(5*time.Second, 5, func() (done bool, err error) {
+	err := retryutil.Retry(10*time.Second, 5, func() (done bool, err error) {
```

Comment:
can we instead retry number instead of interval? increasing interval will slow down our test even if the delection is fast.